packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -255,6 +256,18 @@ class WebView extends StatefulWidget {
   /// By default `debuggingEnabled` is false.
   final bool debuggingEnabled;
 
+  /// The User_Agent used for WebView requests.
+  ///
+  /// By default `userAgent` is null.
+  /// If the `userAgent` is null, the default User Agent from the Android / iOS WebView is used.
```

Comment:
nit: When null the platform's webview default is used for the `User-Agent` header.